lib/core/versions/latest/DeactivateOperation.ts

```diff
@@ -53,11 +53,8 @@ export default class DeactivateOperation implements OperationModel {
       throw new SidetreeError(ErrorCode.DeactivateOperationTypeIncorrect);
     }
 
-    if (typeof operationObject.didSuffix !== 'string') {
-      throw new SidetreeError(ErrorCode.DeactivateOperationMissingOrInvalidDidUniqueSuffix);
-    }
-
-    InputValidator.validateEncodedMultihash(operationObject.revealValue, `deactivate request reveal value`);
+    InputValidator.validateEncodedMultihash(operationObject.didSuffix, 'deactivate request didSuffix');
```

Comment:
I'd suggest to copy the changes to v1.0 folder, this shouldn't affect existing IDs.